### PR TITLE
Makes it so you only see voice in conversation

### PR DESCRIPTION
**Pull request by ynot01**
# General Documentation

### Intent of your Pull Request

Tested on private server

ID no longer shows up in chat when someone is talking
You will only see the voice that you hear

What exactly does this mean?
Chat will no longer display the following:
> Urist (as Captain) says, "Hello, world"
> Urist (as Unknown) says, "Hello, world"

Chat will instead display:
> Urist says, "Hello, world"
> Urist says, "Hello, world"

You will still be able to see their ID by seeing your mouse hover or by examining them, as visual display text has not been changed, only what shows up when someone talks

### Why is this change good for the game?

Runechat exists. The technology is here and this code is from an age where you could not tell where a voice was coming from. You needed this identifying information and now you do not. If you want to match the voice to the face to see if any tomfoolery is about, you need to actually look at the person.

# Changelog

:cl:  
tweak: You can now only see voices in chat
/:cl:

### CODE REVIEW

**Comment by Ratvar** [comment]
I am guessing NTSL is still able to magially show what ID people are wearing, might be inconsistent

**Comment by ynot01** [comment]
> I am guessing NTSL is still able to magially show what ID people are wearing, might be inconsistent

NTSL apparently uses a different variable (according to the comment in the code) so this shouldn't affect it
this is primarily for in-person conversation, I'm not really keen on touching ntsl code

**Comment by ToasterBiome** [comment]
Chameleon mask interaction?

**Comment by alexkar598** [comment]
But why, this is just a low key antag buff because ????.

Do we really want to make antags impersonating sec officers even easier? I dont, antag as sec is boring as hell and steam rolly.

The other thing this allows is for antags to steal the spare but keep their face visible so unless you examine the person and see they're wearing someone else's id, they will remain looking as themselves and they'll speak as themselves too. Bonus points if you hide the Id inside a wallet or pda now it's just impossible to tell.

Tl;dr: Antag buff that encourages stupid metas such as officer impersonation and hiding things in plain sight.

**Comment by ynot01** [comment]
> Chameleon mask interaction?

works as expected, if your voice is changed the changed voice will show up

**Comment by Mqiib** [comment]
This is only an antag buff by way of being able to be a bit less suspicious to unobservant crewmembers. I think it's a fine change and it makes sense, tbh.

**Comment by Tipy1802** [comment]
This makes impersonating security or command even easier. Nobody likes security antags because it’s so fucking overpowered and unfair.
Currently for a traitor to do it they need to spend TC which makes it somewhat fair
And no nobody just examines someone and checks their ID out of the blue

**Comment by ToasterBiome** [comment]
Nobody likes security

**Comment by maxion12345** [comment]
Makes sense. Never did like it when it was super easy to catch antags because I looked in chat and see "janitor (as cmo)"

**Review by jcatjmeow — APPROVED** [review]

**Comment by Darkstick** [comment]
My screen resolution is whack and I can't see runetext at all, unfortunately (it's just a mass of unreadable ugly pixels). This would cripple me and no doubt others.

**Comment by ynot01** [comment]
> My screen resolution is whack and I can't see runetext at all, unfortunately (it's just a mass of unreadable ugly pixels).

sounds like a runetext issue

**Comment by Cartlord** [comment]
> 
> 
> > My screen resolution is whack and I can't see runetext at all, unfortunately (it's just a mass of unreadable ugly pixels).
> 
> sounds like a runetext issue

You can't just say that. You've just brushed off a rather serious problem, that YOUR PR would introduce, and blown it off like it was someone else's job to fix.

**Comment by JamieD1** [comment]
Test Merge

**Review by redmoogle — APPROVED** [review]
Code is fine

**Comment by JamieD1** [comment]
Fuck it lets give it a go

**Comment by JamieD1** [comment]
Going to close, this requires people to be running with rune chat enabled, I cant willingly merge this as I know people that don't have these and they would be at a disadvantage.

**Comment by ynot01** [comment]
understandable
